### PR TITLE
fix(openshift): update Quickstart text/formatting

### PR DESCRIPTION
**Pull request by ebaron**
# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
* [ ] Read the [contributing guidelines](https://github.com/cryostatio/cryostat/blob/main/CONTRIBUTING.md)
* [ ] Linked a relevant issue which this PR resolves
* [ ] Linked any other relevant issues, PR's, or documentation, if any
* [ ] Resolved all conflicts, if any
* [ ] Rebased your branch PR on top of the latest upstream `main` branch
* [ ] Attached at least one of the following labels to the PR: `[chore, ci, docs, feat, fix, test]`
* [ ] [Signed all commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits): `git commit -S -m "YOUR_COMMIT_MESSAGE"`
_______________________________________________

Related to: https://github.com/cryostatio/cryostat-operator/pull/668

## Description of the change:
Fixes indentation and fields. Removes explicit summary of tasks since this is already generated when working through the Quickstart. Removed placeholder text and summary fields. Generally followed the example of our existing quickstart: https://github.com/openshift/console-operator/pull/770

## Motivation for the change:
When installing the bundle with the Quickstart included, I see the following errors:
```
unknown field "spec.tasks[0].review.summary", unknown field "spec.tasks[1].review.summary", unknown field "spec.tasks[2].conclusion", unknown field "spec.title"
```

## How to manually test:
1. `oc create -f config/openshift/cryostat-quickstart.yaml`
2. Access the Quickstart from the Help menu in the OpenShift Console.

### CODE REVIEW

**Review by andrewazores — DISMISSED** [review]

File: config/openshift/cryostat-quickstart.yaml [hunk 1]
```diff
@@ -7,86 +7,116 @@ metadata:
...
+
+        1. In the navigation menu, click **Topology**.
+
+        1. Click on the quarkus-quickstarts deployment name (or your application's service name).
+
+        1. Click on the **Resources** tab.
+  - description: >
+      View your Java application as a target in the Red Hat build of Cryostat web application:
+
+      1. Click on the perspective switcher at the top of the navigation, and select **Developer**.
```
Comment (cmah88) [inline config/openshift/cryostat-quickstart.yaml:98]: Should be " 1. In the main navigation, click the dropdown menu and select **Developer**. "

Comment (ebaron) [reply config/openshift/cryostat-quickstart.yaml]: Fixed

File: config/openshift/cryostat-quickstart.yaml [hunk 2]
```diff
@@ -7,86 +7,116 @@ metadata:
...
+      instructions: >
+        #### Verify that the service port is added to the sample resource:
+
+        1. In the main navigation, click the dropdown menu and select **Developer**.
+
+        1. In the navigation menu, click **Topology**.
+
+        1. Click on the quarkus-quickstarts deployment name (or your application's service name).
+
+        1. Click on the **Resources** tab.
```
Comment (cmah88) [inline config/openshift/cryostat-quickstart.yaml:94]: add a step below:

1. Do you see **Service port: jfr-jmx -> Pod port: 9091** in the quarkus-quickstarts service?

Comment (ebaron) [reply config/openshift/cryostat-quickstart.yaml:94]: Fixed

**Review by cmah88 — APPROVED** [review]
LGTM

**Pull request merged**